docs/development/setting-up-environment.md

```diff
@@ -1,80 +1,186 @@
 # Setting up your development environment
 
-* Fork the repository - [quicklink](https://github.com/pyscript/pyscript/fork)
+These steps will help you set up your development environment. We suggest completing each step before going to the next step, as some parts have dependencies on previous commands.
 
-* Clone your fork of the project
+## Prepare your repository
 
-```
+* Create a fork of the [PyScript github repository](https://github.com/pyscript/pyscript/fork) to your github.
+
+* In your development machine, clone your fork of PyScript. Use this command in your terminal.
+
+```sh
 git clone https://github.com/<your username>/pyscript
 ```
 
-* Add the original project as your upstream (this will allow you to pull the latest changes)
+* With the following command, add the original project as your upstream. This will allow you to pull the latest changes.
 
 ```sh
 git remote add upstream git@github.com:pyscript/pyscript.git
```

Comment:
since we are here, shall we suggest also the HTTPS upstream in case there are access issues to the SSH one?

That is `git remote add upstream https://github.com/pyscript/pyscript.git` as alternative, which should always work.